docs/_running-tests/safari.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: Safari
+---
+To run Safari on macOS, some manual setup is required:
+
+  * Disable "Block pop-up windows" in the security preferences.
```

Comment:
This should be doable with `defaults`. We should be able to set this when we start the browser and reset it to its previous value after running.